foam_controlwrapper/io_utils.py

```diff
@@ -0,0 +1,34 @@
+""" Utility functions for file IO
+
+"""
+import simphonyfoaminterface as foamface
+
+from .foam_mesh import FoamMesh
+
+
+def read_foammesh(name, path):
```

Comment:
I think that this can be also done by creating  the mesh on the fly and read it. I'll handle that also now.